### DISCUSSION TITLE
Carlinkit alternatives

### DISCUSSION THREAD

**Discussion by ThatSmart** [tesla-android/issue-tracker#264] [Q&A]
Are there any alternatives to CarlinKit CPC200-Autokit, CPC200-CCPA? Are there any other suppleres than Carlinkit?

Would the CARLINKIT 4.0 CPC200-CP2A work?

**Comment by HA-TB303** [comment]
I have just tried that exact model, but it does not seem to work.

**Comment by HA-TB303** [comment]
I also got hold of a Carlinkit CPC200-U2W, but I can also not get that one to work.

**Reply by mikegapinski** [reply]
U2W is for OEM wired CarPlay -> wireless carplay.

ONLY CPC200-CCPA works

**Reply by HA-TB303** [reply]
Thanks for clarifying that. Will look for that model then.
